Resources/ServerInfo/Guidebook/Antagonist/Revolutionaries.xml

```diff
@@ -0,0 +1,54 @@
+<Document>
+  # Revolutionaries
+
+  - Revolutionaries are antagonists that are sponsored by the Syndicate to take over the station.
+
+  ## Head Revolutionaries
+  
+<Box>
+  <GuideEntityEmbed Entity="Flash" Caption="Flash"/>
+  <GuideEntityEmbed Entity="ClothingEyesGlassesSunglasses" Caption="Sunglasses"/>
+</Box>
+
+  [color=#5e9cff]Head Revolutionaries[/color] are chosen at the start of the shift and are tasked with taking over the station by killing or exiling all of the Command staff. To aid you in this task you will be given a [color=#a4885c]Flash[/color] and a pair of [color=#a4885c]Sunglasses[/color].
+
+  ## Conversion
+
+  <Box>
+  <GuideEntityEmbed Entity="Flash" Caption="Flash"/>
+</Box>
+
+  The [color=#a4885c]Flash[/color] is used to convert crew members to the side of the revolution. You can convert crew members by using a [color=#a4885c]Flash[/color] in [color=#ff0000]harm mode[/color] and attacking someone with it. Any flash can be used for conversion but remember that flashes have limited charges.
```

Comment:
The second sentence duplicates the first, consider merging.

Missing comma before "but"